File: 连续字符.cpp
class Solution {
public:
    int maxPower(string s) {
        int ans = 1;
        int length = 1;
        char prev = s[0];
        for(int i = 1; i < s.length(); i++){
            if(s[i] == prev)
                length++;
            else{
                ans = max(ans, length);
                if(s.length() - i <= ans)
                    break;
                length = 1;
                prev = s[i];
            }
        }
        ans = max(ans, length);
        return ans;
    }
};

File: 连续字符.py
class Solution(object):
    def maxPower(self, s):
        """
        :type s: str
        :rtype: int
        """
        cnt, ans = 1, 1
        for i in range(1, len(s)): # 不用判断第一个
            if s[i] == s[i-1]:
                cnt += 1;
                ans = max(ans, cnt);
            else:
                cnt = 1;
        return ans;